tools/slang-test/slang-test-main.cpp

```diff
@@ -1784,6 +1784,37 @@ SlangResult innerMain(int argc, char** argv)
     auto vulkanTestCategory = categorySet.add("vulkan", fullTestCategory);
     auto unitTestCatagory = categorySet.add("unit-test", fullTestCategory);
     auto compatibilityIssueCatagory = categorySet.add("compatibility-issue", fullTestCategory);
+    
+#if SLANG_WINDOWS_FAMILY
+    auto windowsCatagory = categorySet.add("windows", fullTestCategory);
+#endif
+
+#if SLANG_UNIX_FAMILY
+    auto unixCatagory = categorySet.add("unix", fullTestCategory);
+#endif
+
+    TestCategory* fxcCategory = nullptr;
+    TestCategory* dxcCategory = nullptr;
+    TestCategory* glslangCategory = nullptr;
+
+    // Might be better if we had an API on slang so we could get what 'pass-through's are available
+    // This works whilst these targets imply the pass-through/backends
+    {
+        SlangSession* session = context.getSession();
+
+        if (SLANG_SUCCEEDED(spSessionCheckCompileTargetSupport(session, SLANG_DXBC)))
```

Comment:
We should probably add a different API like `spSessionCheckPassThroughSupport` that is keyed on the same enumeration as `spSetPassThrough` or whatever it is called. That would let these calls directly use `SLANG_PASS_THROUGH_FXC` etc. instead of inferring from support for a target that we support a particular pass-through compiler.

This matters because in the long run we may deprecate glslang pass-through because we move to emitting SPIR-V directly.

Reply:
Added spSessionCheckPassThroughSupport.